js/ui/components/discoveryFeed.js

```diff
@@ -16,6 +16,7 @@ const DiscoveryFeedIface = '<node> \
 <method name="hide"> \
   <arg type="u" direction="in" name="timestamp"/> \
 </method> \
+<method name="hidden" /> \
```

Comment:
This method might dissapear from here or be renamed at least, depending on the way the comment in https://github.com/endlessm/eos-discovery-feed/pull/108#discussion_r187405937 gets addressed.